lisbon-public-law-editions-full-note.csl

```diff
@@ -0,0 +1,1418 @@
+    <title-short>LPL Editions Full</title-short>
+    <id>http://www.zotero.org/styles/lisbon-public-law-editions-full-note</id>
+    <link href="http://www.zotero.org/styles/lisbon-public-law-editions-full-note" rel="self"/>
+    <author>
+      <name>Made with Brio</name>
+      <email>madewithbrio@gmail.com</email>
+      <uri>madewithbrio.com</uri>
+    </author>
+    <contributor>
+      <name>Lisbon Public Law Editions</name>
+      <email>communication.lpl@fd.ulisboa.pt</email>
+    </contributor>
+    <category citation-format="note"/>
+    <category field="generic-base"/>
+    <summary>The Full Note variant of the Lisbon Public Law Editions</summary>
+    <updated>2022-08-11T18:41:21+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
+    <terms>
+      <term name="editor" form="verb-short">ed.</term>
+      <term name="translator" form="verb-short">trans.</term>
+      <term name="translator" form="short">trans.</term>
+      <term name="editortranslator" form="verb-short">ed. and trans.</term>
+      <term name="editortranslator" form="verb">Edited and translated by</term>
+      <term name="translator" form="short">trans.</term>
+    </terms>
+  </locale>
+  <macro name="editor-translator">
+    <group delimiter=", ">
+      <group delimiter=" ">
+        <choose>
+          <if variable="container-author reviewed-author" match="any">
+            <group>
+              <names variable="container-author reviewed-author">
+                <label form="verb-short" text-case="lowercase" suffix=" "/>
+                <name and="text" delimiter=", "/>
+              </names>
+            </group>
+          </if>
+        </choose>
+      </group>
+      <names variable="editor translator" delimiter=". ">
+        <name et-al-min="1" et-al-use-first="0" initialize-with="" name-as-sort-order="first" sort-separator=" "/>
+        <label text-case="lowercase" prefix=", " suffix="."/>
+      </names>
+    </group>
+  </macro>
+  <macro name="secondary-contributors-note">
+    <choose>
+      <if type="chapter entry-dictionary entry-encyclopedia paper-conference" match="none">
+        <text macro="editor-translator"/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="container-contributors-note">
+    <choose>
+      <if type="chapter entry-dictionary entry-encyclopedia paper-conference" match="any">
+        <text macro="editor-translator"/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="secondary-contributors">
+    <choose>
+      <if type="chapter entry-dictionary entry-encyclopedia paper-conference" match="none">
+        <names variable="editor" delimiter=". ">
+          <name et-al-min="10" et-al-use-first="1" et-al-subsequent-min="10" et-al-subsequent-use-first="1" initialize-with="" name-as-sort-order="first" sort-separator=" "/>
```

Comment:
remove the subsequent conditions here (since they're the same as the original) -- generally, I have  a preference for setting et al on the citation and bibliography element rather than individually for creators, where it's hard to spot.